attachment_metadata/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# @ 2015 Valentin CHEMIERE @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{'name': 'attachment_metadata',
+ 'version': '0.0.1',
```

Comment:
The version scheme is not right. `version`should be 8.0.1.0.0 (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)